README.md

```diff
@@ -1,5 +1,9 @@
+[![Build Status](https://travis-ci.org/PenneyGadget/lookingfor.svg?branch=master)](https://travis-ci.org/PenneyGadget/lookingfor)
```

Comment:
I still don't see the LookingFor repo in my own Travis account. It would probably be better if it was linked to yours anyway, no?